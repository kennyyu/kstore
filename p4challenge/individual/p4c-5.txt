u_delete=select(ua,100000,200000)
delete(u_delete,ua,ub,uc,ud)
tb_select=select(tb,1000)
ta_fetch=fetch(ta,tb_select)
ud_select=select(ud,9000,9000000)
ua_fetch=fetch(ua,ud_select)
t_id,u_id=hashjoin(ta_fetch,ua_fetch)
td_fetch=fetch(td,t_id)
td_avg=avg(td_fetch)
uc_fetch=fetch(uc,u_id)
uc_avg=avg(uc_fetch)
tuple(td_avg,uc_avg)
