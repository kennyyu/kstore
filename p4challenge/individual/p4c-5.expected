(618527,238469)
